src/app/plugins/platform/Docker/plugins/cms/Drupal8/installDrupal.ts

```diff
@@ -120,4 +114,84 @@ async function installDrupal({
   }
 }
 
+/**
+ * Rename the generated web root if necessary.
+ *
+ * @param {InstallDrupalOptions} {
+ *     serviceDirectory,
+ *     documentRoot,
+ *   }
+ * @returns {Promise<string>} - The path to the Drupal installation root.
+ */
+async function renameWebRoot({
```

Comment:
:thinking: does this function need to return `drupalRoot`? It seems like this function shouldn't return anything since it's mainly here for side effects. I know it's needed by other functions, but this return value isn't really part of `renameWebRoot`'s responsibility - it's an implementation detail of needing the full path in order to do filesystem operations.